tests/Http/PostTest.php

```diff
@@ -1169,21 +1169,21 @@ public function testPostsIndexAsVoterRegistrationReferrer()
         $firstCompletedVoterRegReferral = factory(Post::class)->states('voter-reg', 'register-form')->create(['referrer_user_id' => $referrerUserId]);
         $secondCompletedVoterRegReferral = factory(Post::class)->states('voter-reg', 'register-OVR')->create(['referrer_user_id' => $referrerUserId]);
 
-        // Add a non-completed voter reg referral for our referrer, which shouldn't be visible.
+        // Add a non-completed voter reg referral for our referrer, which should be visible.
         factory(Post::class)->states('voter-reg', 'step-1')->create(['referrer_user_id' => $referrerUserId]);
 
-        // Add a completed voter reg without a referrer, which shouldn't be visible.
+        // Add a completed voter reg without a referrer, which should be visible.
         factory(Post::class)->states('voter-reg', 'register-OVR')->create();
 
-        // Add a completed voter reg referral for a different referrer, which shouldn't be visible.
-        factory(Post::class)->states('voter-reg', 'register-form')->create(['referrer_user_id' => $this->faker->unique()->northstar_id]);
+        // Add pending referrals for a different referrer, which shouldn't be visible.
+        factory(Post::class)->states('photo', 'pending')->create(['referrer_user_id' => $this->faker->unique()->northstar_id]);
 
         // Add a pending photo post, which shouldn't be visible.
         factory(Post::class)->states('photo', 'pending')->create();
 
         $response = $this->withAccessToken($referrerUserId)->getJson('api/v3/posts');
         $response->assertStatus(200);
-        $response->assertJsonCount(2, 'data');
+        $response->assertJsonCount(4, 'data');
```

Comment:
There are checks below for `register-form` and `register_OVR` showing up, should we add one for `step-1`?

Reply:
Updated, good call!